src/sentences/utils.rs

```diff
@@ -1,7 +1,6 @@
-use core::str;
-
 use arrayvec::ArrayString;
 use chrono::{NaiveDate, NaiveTime};
+use core::str;
```

Comment:
This change was not needed.
Also it would be nice to keep `core/std/alloc` separate from other imports.

Reply:
Oh yeah, that's probably `rustfmt` doing its thing :sweat_smile:.
Regarding the use of `core/std/alloc`, I'll move the `number_in_range` parser to `utils` to avoid spreading the import of `core`